frontend/AnimeCore/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,279 @@
+				      type="button"
+				      class="btn bg-primary-500 font-bold text-white md:h-[4.3vw] md:w-[6.75vw] md:rounded-[0.625vw] md:text-[0.87vw]"
+					     >
+					      <div class="flex items-center justify-center md:gap-[0.7vw]">
+					      	  <PlayCircle width="25" height="25" color="white" />
+						      <div class="flex flex-col items-start">
+							       <span class="md:leading-5">Watch</span>
+							       <span class="font-normal leading-none text-surface-100 md:text-[0.625vw]">Ep 01</span>
+						      </div>
+					      </div>
+				    </button>
+
+				    {#each Object.entries(icon_mapping.anime_options) as item}
+					    {@const item_name = item[0]}
+					    {@const item_icon = item[1].icon}
+
+					    {@const component = item_icon.component}
+					    {@const component_width = item_icon.width}
+					    {@const component_height = item_icon.height}
+					    {@const component_color = item_icon.color}
+
+					    <button
+					       type="button"
+					       class="btn bg-secondary-100 capitalize text-surface-500 md:h-[4.3vw] md:w-[4.3vw] md:rounded-[0.625vw]
+					       md:text-[0.87vw] md:font-semibold"
+					    >
+					       <div class="flex flex-col items-center justify-center md:gap-[0.68vw]">
+						       	<svelte:component
+						            this={component}
+						        	width={component_width}
+						        	height={component_height}
+						        	color={component_color}
+						       />
+						       <span class="md:leading-[1vw]">{item_name}</span>
+					       </div>
+					    </button>
+				    {/each}
+				</div>
+
+				<div class="md:mt-[1.25vw] flex md:gap-[0.625vw]">
+					{#each Object.entries(icon_mapping.user_options_icons) as item}
+						{@const item_icon = item[1].icon}
+
+						{@const component = item_icon.component}
+						{@const component_width = item_icon.width}
+						{@const component_height = item_icon.height}
+						{@const component_variant = item_icon.variant}
+
+						<button
+							type="button"
+							class="btn-icon bg-warning-400 p-0 text-surface-500 md:w-[1.875vw] md:rounded-[0.25vw]"
+						>
+							<svelte:component
+								this={component}
+								width={component_width}
+								height={component_height}
+								variant={component_variant}
+							/>
+						</button>
+					{/each}
+				</div>
+			</div>
+		</div>
+
+		<div class="col-span-5 pr-8 md:w-[26.625vw]">
+			<div class="flex items-center gap-4">
+				<span class="font-semibold md:text-[1.25vw] md:leading-[1.5vw]"> Synopsis </span>
+				<SettingsOutline width="11" height="11" class="opacity-75" />
+			</div>
+
+			<ScrollArea
+				bind:scroll_top
+				offsetScrollbar
+				parentClass="mt-5"
+				class="text-justify md:max-h-[9.5vw] md:text-[0.75vw] md:leading-[1vw]"
+			>
+				<div
+					class="{scroll_top > 30 ? 'bg-gradient-to-t' : 'bg-gradient-to-b'} from-surface-50 to-surface-50/50 bg-clip-text text-transparent duration-300"
+				>
+					{anime.description}
+				</div>
+			</ScrollArea>
```

Comment:
I found a bug. So when scroll is in middle it should have both `bg-gradient-to-t` and `bg-gradient-to-b`

Reply:
> I found a bug. So when scroll is in middle it should have both `bg-gradient-to-t` and `bg-gradient-to-b`

Is that necessary? since it was just jumbing to top and bottom?

Reply:
Eye Candi help ape

Reply:
Yea, It'll be helpful for someone blind like you XD
Jocking aside I'll fix that on next commit  :)